package/base-files/files/lib/upgrade/mmc.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+. /lib/functions.sh
+
+mmc_do_upgrade() {
+
+    command -v losetup >/dev/null || {
```

Comment:
broken indent here. I would also remove the empty line before.

Reply:
Fixed, thanks!